src/com/sun/ts/tests/jaxrs/platform/managedbean299/TSAppConfig.java

```diff
@@ -19,11 +19,11 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import jakarta.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
-import jakarta.ws.rs.core.Application;
-import jakarta.ws.rs.core.Context;
-import jakarta.ws.rs.core.UriInfo;
+import javax.annotation.PostConstruct;
```

Comment:
jakarta.annotation and jakarta.ws were reverted to resolve conflicts ?

Reply:
thanks, will figure out what happened and fix

Reply:
yes, I did that when I was resolving the 3 file conflicts, oops, fixing...